### PR TITLE
`Paywalls`: template 5

### PR DESCRIPTION
**Pull request by NachoSoto**
image

### CODE REVIEW

File: RevenueCatUI/Templates/Template3View.swift [hunk 1]
```diff
@@ -129,8 +129,7 @@ private struct FeatureView: View {
     private var icon: some View {
         Circle()
             .overlay {
-                if let iconName = self.feature.iconID,
-                   let icon = PaywallIcon(rawValue: iconName) {
+                if let icon = self.feature.icon {
```
Comment (NachoSoto) [inline RevenueCatUI/Templates/Template3View.swift:132]: Extracted this into an extension to reuse it.

File: RevenueCatUI/Templates/Template2View.swift [hunk 2]
```diff
@@ -264,7 +263,7 @@ struct Template2View: TemplateViewType {
     private var iconSize: CGFloat = 140
 
     private static let fadedColorOpacity: CGFloat = 0.3
-    private static let cornerRadius: CGFloat = 15
+    private static let cornerRadius: CGFloat = Constants.defaultPackageCornerRadius
```
Comment (NachoSoto) [inline RevenueCatUI/Templates/Template2View.swift:266]: Extracted to have common values.

File: RevenueCatUI/Templates/Template2View.swift [hunk 3]
```diff
@@ -117,7 +117,6 @@ struct Template2View: TemplateViewType {
                     self.selectedPackage = package
                 } label: {
                     self.packageButton(package, selected: isSelected)
-                        .contentShape(Rectangle())
```
Comment (NachoSoto) [inline RevenueCatUI/Templates/Template2View.swift:120]: Extracted to `PackageButtonStyle`.

**Comment by NachoSoto** [comment]
Gotta update snapshots

File: RevenueCatUI/Templates/Template5View.swift
Comment (joshdholtz) [inline RevenueCatUI/Templates/Template5View.swift:1]: This is an intense view but I love it! 😍

**Review by joshdholtz — APPROVED** [review]

**Pull request merged**